tests/service/test_service_without_skip_host_check.yml

```diff
@@ -24,22 +24,22 @@
       ok_as_delegate: no
       ok_to_auth_as_delegate: no
     register: result
-    failed_when: not result.changed
+    failed_when: not result.changed or result.failed
```

Comment:
This change is unrelated.

Reply:
This change is in a TEMP commit, so service tests pass.